lib/chef/resource/mount.rb

```diff
@@ -24,8 +24,6 @@ class Resource
     class Mount < Chef::Resource
       description "Use the mount resource to manage a mounted file system."
 
-      identity_attr :device
-
       state_attrs :mount_point, :device_type, :fstype, :username, :password, :domain
```

Comment:
if we move to properties you might as well nuke this all since everything is a state attrs by default